src/main/java/org/bitbuckets/drive/controlsds/DriveControlSDS.java

```diff
@@ -91,7 +92,7 @@ void guaranteedLoggingLoop() {
         });
     }
 
-    public double[] getGyroXYZ_mps() {
+    public double[] getGyroXYZ_dps() {
```

Comment:
How noisy is this? Would it benefit from some sort of filter?

Reply:
it was not noisy, it just looked noisy at the time because I was reading the wrong value lol